mteb/tasks/Retrieval/multilingual/MIRACLRetrieval.py

```diff
@@ -7,83 +7,133 @@
 from ....abstasks import MultilingualTask
```

Comment:
Can you please run evaluations on `intfloat/multilingual-e5-small` and `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2` ?